src/status_im/wallet/choose_recipient/core.cljs

```diff
@@ -72,6 +73,13 @@
  (fn [{:keys [registry ens-name cb]}]
    (ens/get-addr registry ens-name cb)))
 
+(fx/defn resolve-token-address-from-qr
+  {:events [:wallet.send/resolve-token-address-from-qr]}
+  [{:keys [db]} address]
+  (let [raw-data "ethereum:pay-snt.thetoken.eth/transfer?address=acolytec3.stateofus.eth&uint256=1e2"
```

Comment:
Is this in the main code? please undo this.

Reply:
is it the place you said you had to hardcode @acolytec3 ?

Reply:
Yep, was having trouble passing the complete URL string through all the functions and then the callback so had just hardcoded it to see if the general proof of concept worked.

Reply:
Fixed this one based on the below feedback from @yenda.